### PR TITLE
Sort rooms by last room usage.

### LINKED ISSUE #68: Sort by last room usage

### PR DESCRIPTION
**Pull request by Ivansss**
Fixes #68

### CODE REVIEW

File: js/models/roomcollection.js [hunk 1]
```diff
@@ -28,7 +28,9 @@
 
 	var RoomCollection = Backbone.Collection.extend({
 		model: OCA.SpreedMe.Models.Room,
-		comparator: 'name',
+		comparator: function(model) {
+			return -(model.get('lastPing'));
```
Comment (ChristophWurst) [inline js/models/roomcollection.js]: AFAIK 'lastPing' is never set on room objects, or at least I could not find any code that would set it

Comment (Ivansss) [inline js/models/roomcollection.js]: When we 'ping' the current room, a timestamp is set in 'lastPing'
https://github.com/nextcloud/spreed/blob/master/lib/Controller/ApiController.php#L280

Comment (ChristophWurst) [inline js/models/roomcollection.js]: That updates the room participant table ``spreedme_room_participants``. That rooms table does not have that attribute, does it?

(I have not really tested your changes yet, but as far as I know the code, this won't work)

Comment (Ivansss) [inline js/models/roomcollection.js]: Yes, that's right. That attribute is part of `spreedme_room_participants` table.

I'm kind of new with Backbone, so I was just checking what we receive when we do `GET` on `/apps/spreed/api/room` endpoint and assuming all these attributes we receive are part of the RoomCollection Model.

Comment (Ivansss) [inline js/models/roomcollection.js]: The thing is that rooms get sorted by last room usage with the code above (maybe it's just magic 😄 ).

Comment (LukasReschke) [inline js/models/roomcollection.js]: The lastPing is already returned, that's what the join does ;) 

The return of `/apps/spreed/api/room` is:

```
[
   {
      "id":"26",
      "name":"test",
      "type":"1",
      "userId":"admin",
      "roomId":"26",
      "lastPing":"1476883919",
      "displayName":"test",
      "validRoom":true,
      "count":0
   },
   {
      "id":"27",
      "name":"test1",
      "type":"1",
      "userId":"admin",
      "roomId":"27",
      "lastPing":"1476883922",
      "displayName":"Dick",
      "validRoom":true,
      "count":0
   }
]
```

**Comment by ChristophWurst** [comment]
well, then it's fine by me, just small linting errors and the commit is not signed off 👍

File: js/models/roomcollection.js (hunk 1 above)
Comment (jancborchardt) [inline js/models/roomcollection.js]: The avatar is Dick Cheney ;)

**Pull request merged**